### DISCUSSION TITLE
Set or infer conda environment

### DISCUSSION THREAD

**Discussion by raghavmallampalli** [neoclide/coc.nvim#3256] [Q&A] [answered]
I use conda frequently. Is there a way to set or infer the conda env from the host shell? In my experience, it picks one of the installed environments and uses that for code completion in every successive instance of vim.

If there was a command I could run to change the environment or to set it while entering vim it would be convenient.

**Comment by chemzqm** [comment, marked as answer]
coc.nvim knows nothing about language specific env.  Ask the maintainer of python extension or language server you're using
